code/game/objects/items/devices/megaphone.dm

```diff
@@ -27,7 +27,7 @@
 		return
 
 	var/message = copytext(sanitize(input(user, "Shout a message?", "Megaphone", null)  as text),1,MAX_MESSAGE_LEN)
-	if(!message)
+	if(!message || message == "")
```

Comment:
Eh maybe !message alone works, I never bothered finding out.
Op 9-jun.-2015 04:21 schreef "Cheridan" <notifications@github.com>:

> In code/game/objects/items/devices/megaphone.dm
> <https://github.com/tgstation/-tg-station/pull/9837#discussion_r31978416>:
>
> > @@ -27,7 +27,7 @@
> >  		return
> >
> >  	var/message = copytext(sanitize(input(user, "Shout a message?", "Megaphone", null)  as text),1,MAX_MESSAGE_LEN)
> > -	if(!message)
> > +	if(!message || message == "")
>
> if this kind of check is called elsewhere commonly it might be good to
> turn this into a helper proc
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/9837/files#r31978416>.
>